frontend/src/lib/Buttons.ts

```diff
@@ -654,6 +702,102 @@ export default class Buttons {
       callback(unsuccessfulIds, !unsuccessfulIds.length);
     }
   }
+
+  private _dialogActionHandlerMultiId(
```

Comment:
nit: This method seems to be mostly duplicate code with the existing method `_dialogActionHandler`, and similarly for the next one `_dialogClosedMultiId`. Would it be possible to reduce the duplication by merging the two? Say you can extend the existing method to accept multiple Ids, the existing method would only use the first one and ignore the second value which would be empty/nil for them. And the newly added `_deletePipelineVersionV2` would take advantage of both Ids. WDYT?

Reply:
Merged!